.github/workflows/pkgdown.yaml

```diff
@@ -34,19 +35,29 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::pkgdown, local::.
+          extra-packages: any::pkgdown, local::., any::cli
           needs: website
 
       - name: Build site
-        env:
-          DELPHI_EPIDATA_KEY: ${{ secrets.SECRET_EPIPROCESS_GHACTIONS_DELPHI_EPIDATA_KEY }}
         run: |
-          if (startsWith("${{ github.event_name }}", "pull_request")) {
-              mode <- ifelse("${{ github.base_ref }}" == "main", "release", "devel")
+          override <- if (startsWith("${{ github.event_name }}", "pull_request")) {
+              if ("${{ github.base_ref }}" == "main") {
+                list(development = list(mode = "release", version_label = "light"))
+              } else {
+                list(development = list(mode = "devel", version_label = "success"))
+              }
           } else {
-              mode <- ifelse("${{ github.ref_name }}" == "main", "release", "devel")
+              if ("${{ github.ref_name }}" == "main") {
+                list(development = list(mode = "release", version_label = "light"))
+              } else {
```

Comment:
These are CSS keywords from the [Bootswatch: Cosmo theme here](https://bootswatch.com/cosmo/).

Reply:
I think we don't really need to worry about the non-main/dev case, since this action is only run on `main` and `dev`? But yea the logic here is a bit tangly.

Reply:
Idea was to guard against "" (from referencing the wrong "ref" variable somehow, or some weird series of events triggering github to call something in a new way), and also future-proofing if we add branches other than main & dev but forget to update here.

Reply:
Is that CSS triggering how you want on epidatr right now?  For me, main & dev are both red banners; only difference I see is that dev has a green version number that's a little hard to read, plus a "Dev Status" section on the sidebard.

Reply:
The theme is working as intended; the "success" color is green from the Cosmo theme (look at the text under emphasis class). We struggled to find an emphasis class (ie color) that actually showed up well against the red that wasn't just plain white, which we're trying to avoid to give some clear indication that `dev!=main`.

Reply:
Right, we're using Bootswatch: Cosmo theme, but then redefining a few colors (like "Primary") in _pkgdown.yaml. The default setting of "danger" is very hard to see against our new Primary (CMU red). So are all the other ~black adjacent colors, purple, light blue. So green was the only one left.

Reply:
I'll register my objection to green as well. (also terrible for color blindness). I'd almost prefer black or the original primary blue. Do you lose access to it when you redefine `$primary`?

Reply:
oh, good call on the color blindness! the primary blue was one of our first thoughts, but yeah, redefining `$primary` eliminated it as an option

Reply:
Hacked around the primary by just changing the color of `success`; I took a look at the blue of the original primary, but on substitution it was also hard to read. I chose a much brighter shade of green using https://paletton.com/index.html#uid=75w150kt3qeoVFLrqvVsHjWvJa+, and checked for colorblindness; it should be distinguishable for all viewers, and hopefully the brighter color helps make it more legible. If you want to substitute a particular color though, just replace the definition of `success` in `_pkgdown.yml`

Reply:
![image](https://github.com/cmu-delphi/epiprocess/assets/7095649/11688621-628d-4af0-bc8c-dfb34b13f928)
current color

Reply:
previous green
![image](https://github.com/cmu-delphi/epiprocess/assets/7095649/c7e4a7ea-46ca-462c-9f48-7f1e07eed102)

and the blue
![image](https://github.com/cmu-delphi/epiprocess/assets/7095649/8f91ce1c-8877-4b40-8fac-954e1048c6ab)

Reply:
That (current color) looks pretty good. (I'm not sure any of these are as good as the default all white on main and red version on dev, but I think we want some CMU/Delphi Branding.)

Next up, hex logos.

Reply:
Are there more color changes we want to make here?